plugins/src/webrtcsink/imp.rs

```diff
@@ -524,9 +524,8 @@ impl VideoEncoder {
             _ => unreachable!(),
         }
 
-        let mut s = self
-            .filter
-            .property::<gst::Caps>("caps")
+        let current_caps = self.filter.property::<gst::Caps>("caps");
+        let mut s = current_caps.clone()
```

Comment:
```suggestion
        let current_caps = self.filter.property::<gst::Caps>("caps");
        let mut s = current_caps
```

That cloning shouldn't be necessary as you're already getting a copy of the structure anyway. This probably even gives a clippy warning or not?

Reply:
ping @thiblahute :)